### PR TITLE
DM-20768: Demonstration of script reference documentation

### PR DESCRIPTION
**Pull request by jonathansick**
This demos the new system for documenting the stack's ad hoc scripts proposed in https://sqr-032.lsst.io.

Draft documentation pages:

- [lsst.verify](https://pipelines.lsst.io/builds/937/modules/lsst.verify/index.html)
- [dispatch_verify.py docs](https://pipelines.lsst.io/builds/937/modules/lsst.verify/scripts/dispatch_verify.py.html)
- [inspect_job.py](https://pipelines.lsst.io/builds/937/modules/lsst.verify/scripts/inspect_job.py.html)
- [lint_metrics.py](https://pipelines.lsst.io/builds/937/modules/lsst.verify/scripts/lint_metrics.py.html)
- [Reviewing verification JSON outputs on the command line with inspect_job.py](https://pipelines.lsst.io/builds/937/modules/lsst.verify/reviewing-verification-json-on-the-command-line.html)

See also:

- https://github.com/lsst/templates/pull/38
- https://github.com/lsst-dm/dm_dev_guide/pull/339

### CODE REVIEW

File: doc/lsst.verify/reviewing-verification-json-on-the-command-line.rst [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+   See :doc:`scripts/inspect_job.py` for a complete reference of that script's command-line interface.
+
+.. _SQR-019: https://sqr-019.lsst.io
+
+Running inspect_job.py
+======================
+
+|inspect_job| takes one or more verification framework JSON files as command-line arguments.
+Since JSON files output by the verification framework generally have names that end with :file:`verify.json`, you can quickly inspect all available outputs by using a ``*.verify.json`` wildcard:
```
Comment (SimonKrughoff) [inline doc/lsst.verify/reviewing-verification-json-on-the-command-line.rst]: Above you say `*verify.json`.  That seems safer than `*.verify.json`, but maybe I'm not well informed on how these files are generated.

File: python/lsst/verify/bin/inspectjob.py [hunk 2]
```diff
@@ -19,16 +19,17 @@
...
+"""Print the contents of an lsst.verify.Job file.
 
-This script takes as arguments one or more Job .json files, and prints the
-top-level metadata and a summary of any measurements. It does not print
-metrics that don't have measurements (there are far too many) or
-specifications (which are not helpful for testing measurement code).
+This script takes as arguments one or more lsst.verify JSON files, and prints
+the top-level metadata and a summary of any measurements.
+
+It does not print metrics that don't have measurements or specifications.
```
Comment (SimonKrughoff) [inline python/lsst/verify/bin/inspectjob.py]: From the other docs, it sounded like this doesn't care about specifications.  Is that not true?

File: python/lsst/verify/bin/inspectjob.py [hunk 3]
```diff
@@ -110,6 +111,27 @@ def inspect_job(job):
...
+    """
+    parser = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        epilog='More information is available at https://pipelines.lsst.io.')
+    parser.add_argument(
+        'json_paths',
+        nargs='+',
+        metavar='json',
+        help='Verification job JSON file, or files.')
```
Comment (SimonKrughoff) [inline python/lsst/verify/bin/inspectjob.py]: I like the wording used in the comment block of this file:
"lsst.verify JSON file, or files"

File: python/lsst/verify/bin/inspectjob.py (hunk 2 above)
Comment (jonathansick) [reply python/lsst/verify/bin/inspectjob.py]: Aha, I misread the original documentation. I should be that it just doesn't print specifications at all.

**Pull request merged**